department-of-veterans-affairs/abd-vro
# Filter data (possibly PII) from Avro GenericRecord object

## What was the problem?
Previous logging statement was removed in `svc-bie-kafka` kafka consumer code, but camel is still capturing it. We need to tidy up the data immediately after getting Kafka messages to remove any potential PII data exposure.

Associated tickets or Slack threads:
NA

## How does this fix it?[^1]
Create array that only include fields we want to display before logging and sending it to rabbitmq queues

## How to test this PR
- deploy and verify the log


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
